.bazelrc

```diff
@@ -75,12 +75,22 @@ build:linux --action_env=BAZEL_DO_NOT_DETECT_CPP_TOOLCHAIN=1
 build:clang-base --//bazel/cc_toolchains:compiler=clang
 build:clang-base --//bazel/cc_toolchains:libc_version=glibc_host
 
+# We build our own chroot for the sysroot tests, which doesn't work well under bazel's sandbox.
+build:sysroot_common --//bazel/test_runners:test_runner=sysroot_chroot
+test:sysroot_common --strategy TestRunner=standalone
+test:sysroot_common --run_under="bazel/test_runners/sysroot_chroot/test_runner.sh"
+
 build:x86_64_sysroot --config=clang
 build:x86_64_sysroot --//bazel/cc_toolchains:libc_version=glibc2_36
+build:x86_64_sysroot --config=sysroot_common
 
 build:aarch64_sysroot --config=clang
 build:aarch64_sysroot --//bazel/cc_toolchains:libc_version=glibc2_36
 build:aarch64_sysroot --platforms=//bazel/cc_toolchains:linux-aarch64
+build:aarch64_sysroot --config=sysroot_common
+# Increase test timeouts for qemu (don't increase the slowest ones because those are already very long).
+test:aarch64_sysroot --test_timeout=180,600,1800,3600
+test:aarch64_sysroot --test_env=QEMU_STRACE
```

Comment:
should we remove this?

Reply:
This allows people to set `QEMU_STRACE` to cause the test to dump strace logs, but if you don't set it in your env then it doesn't print the strace logs.